packages/sdk/src/constants/abi/develop/HolographTreasury.ts

```diff
@@ -190,6 +190,26 @@ export default narrow([
     stateMutability: 'nonpayable',
     type: 'function',
   },
+  {
```

Comment:
Maybe it would be nice to update the testnet, mainnet and experimental ABIs as well

Reply:
Good idea. @BravoNatalie The ABI for treasury has been updated and can be found in the abis directory of this branch https://github.com/holographxyz/holograph-protocol/pull/228